src/panels/config/logs/error-log-card.ts

```diff
@@ -15,29 +15,20 @@ import { HomeAssistant } from "../../../types";
 class ErrorLogCard extends LitElement {
   @property({ attribute: false }) public hass!: HomeAssistant;
 
-  @internalProperty() private _errorLog?: string;
+  @internalProperty() private _errorHTML!: TemplateResult[];
```

Comment:
```suggestion
  @internalProperty() private _errorHTML!: TemplateResult[] | string;
```

Reply:
Still have to get used to the fact that in TS a variable can have multiple types 🙄 ...